# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../shadow/nullok.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- shadow-4.0.13/etc/pam.d/login.vanilla	2005-11-06 23:52:26.000000000 +0100
+++ shadow-4.0.13/etc/pam.d/login	2005-11-06 23:52:32.000000000 +0100
@@ -1,7 +1,7 @@
 #%PAM-1.0
 # /etc/pam.d/login
 auth            required        pam_securetty.so
-auth            required        pam_unix.so
+auth            required        pam_unix.so nullok
 auth            required        pam_tally.so file=/var/log/faillog onerr=succeed no_magic_root
 auth            required        pam_shells.so
 auth            required        pam_nologin.so
